src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupProjectsTable.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
```

Comment:
File comments

Reply:
Updated in 57a0262c0e02335402f91d1bf8cbb06e421b4c7b